lib/build-tokens.js

```diff
@@ -28,7 +33,20 @@ async function buildTokensCommand(commandArgs) {
     source: tokensSource,
     'source-tokens-only': hasSourceTokensOnly,
     themes,
-  } = minimist(commandArgs, { alias, default: defaultParams, boolean: 'source-tokens-only' });
+    'base-theme': baseTheme,
+    'all-themes': allThemes,
+  } = minimist(commandArgs, { alias, default: defaultParams, boolean: ['source-tokens-only', 'all-themes'] });
+
+  let themesToProcess = null;
+
+  if (allThemes) {
+    themesToProcess = fs
+      .readdirSync(`${tokensSource}/themes/`, { withFileTypes: true })
```

Comment:
This line fails for me locally when I triy to build all themes for Paragon, because `tokensSource` is `undefined` by default. I think this setting is used to overwrite default tokens which we don't need to do in Paragon, so we don't use this parameter internally in repository.

Changing this line to ``.readdirSync(`${tokensSource || path.resolve(__dirname, '../tokens/src')}}/themes/`, { withFileTypes: true })`` seems to work for me (basically defaulting to the root folder of Paragon's tokens)

Reply:
I've applied your suggestion. Do tell if there are any other changed you'd like me to make to get this merged.

Reply:
@xitij2000 It seems we still have problems with the `tokensSource`. Could you check if everything is working correctly?